pom.xml

```diff
@@ -15,6 +15,10 @@
         <version.lucene>5.5.5</version.lucene>
         <!-- This should match the major.minor of the Lucene version, optionally revisioned with module micro fixes -->
         <slot.simple.id>5.5</slot.simple.id>
+        <slot.full.id>${version.lucene}</slot.full.id>
+        <!-- Needs to match the target WildFly server version -->
+        <version.org.wildfly.core>3.0.8.Final</version.org.wildfly.core>
```

Comment:
Seems to me `3.0.8.Final` does not match "the target WildFly server version", which is 11 something... ?
Maybe rather "Needs to match the version of wildfly-core used in the target WildFly server version"?

Reply:
ok I'll change it. I thought it would be clear, if it had to match "verbatim" I wouldn't have a second parameter ;)

BTW I had started looking into a way to import the right BOM to automate this but there things I didn't like in the outcome. Just saying this comment is hopefully not going to live too long.